stdlib/function.ncl

```diff
@@ -2,74 +2,85 @@
   function = {
     id : forall a. a -> a
     | doc m%"
-    The identity function, that is
-    ```nickel
-    id x == x
-    ```
-    for any value `x`.
-    "%
+        The identity function, that is
+        ```nickel
+        id x = x
+        ```
+        for any value `x`.
```

Comment:
I think a new line before (but the after, I'm not sure) a fenced code block is advised by markdownlint. I wonder if we could even run markdownlint on the in-code documentation as part of the CI, via nickel-doc

```suggestion
        The identity function, that is
        
        ```nickel
        id x = x
        ```

        for any value `x`.
```